tests/metrics/README.md

```diff
@@ -4,19 +4,21 @@
 Some tests require you to enable the debug mode of cc-oci-runtime. To enable it, please take a look at:
 https://github.com/01org/cc-oci-runtime#18running-under-docker
 
-To run the metrics, just run:
+
+### Run all metrics tests
+To run the metrics, please use Makefile rule `metrics-tests` on the top level of this repository:
 
 ```bash
-$ cd tests/metrics
-$ ./run_docker_metrics
+$ sudo -E make metrics-tests
```

Comment:
Ah, 'sudo -E' - that feels better. Oddly, it still doesn't actually work for me (even though a `sudo -e echo $PATH` shows the go binary path in there???). I think we can run with this as is for the minute though - it is a step in the right direction.